packages/ui/core-components/src/lib/organisms/layout/EvidenceDefaultLayout.svelte

```diff
@@ -52,38 +59,51 @@
 
 <div data-sveltekit-preload-data={prefetchStrategy} class="antialiased text-gray-900">
 	<ErrorOverlay />
-	<Header
-		bind:mobileSidebarOpen
-		{title}
-		{logo}
-		{neverShowQueries}
-		{fullWidth}
-		{maxWidth}
-		{hideSidebar}
-		{githubRepo}
-		{slackCommunity}
-		{xProfile}
-		{algolia}
-	/>
+	{#if !hideHeader}
+		<Header
+			bind:mobileSidebarOpen
+			{title}
+			{logo}
+			{neverShowQueries}
+			{fullWidth}
+			{maxWidth}
+			{hideSidebar}
+			{githubRepo}
+			{slackCommunity}
+			{xProfile}
+			{algolia}
+		/>
+	{/if}
 	<div
 		class={(fullWidth ? 'max-w-full ' : maxWidth ? '' : ' max-w-7xl ') +
 			'print:w-[650px] mx-auto print:md:px-0 print:px-0 px-6 sm:px-8 md:px-12 flex justify-start'}
 		style="max-width:{maxWidth}px;"
 	>
 		{#if !hideSidebar}
 			<div class="print:hidden">
-				<Sidebar {fileTree} bind:mobileSidebarOpen {title} {logo} {builtWithEvidence} />
+				<Sidebar
+					{fileTree}
+					bind:mobileSidebarOpen
+					{title}
+					{logo}
+					{builtWithEvidence}
+					{hideHeader}
+				/>
 			</div>
 		{/if}
 		<main
-			class={(!hideSidebar ? 'md:px-8 ' : '') +
-				'flex-grow overflow-x-hidden print:px-0 print:md:px-0 py-8'}
+			class={(!hideSidebar ? 'md:pl-8 ' : '') +
+				(!hideTOC ? 'md:pr-8 ' : '') +
+				(!hideHeader ? 'mt-[74px] ' : 'mt-[26px] ') +
```

Comment:
Could we move the existing breadcrumb behaviour here to avoid having to hardcode pixels? 
 
```suggestion
				(!hideHeader ? ' mt-16 sm:mt-20 ' : ' mt-8 sm:mt-12 ') +

```

Reply:
I found that it was just between `mt-20` and `mt-18` to get the page title aligned with the top of the sidebar. This is with `mt-20`:

![CleanShot 2024-03-27 at 12 01 22@2x](https://github.com/evidence-dev/evidence/assets/12602440/4c6b45df-7bfb-46b9-bf1b-b0569193585d)

Reply:
I've moved the breadcrumb logic here - used tailwind sizes wherever possible, and adjusted for mobile. It should match up with what we've got now